NuGet.config

```diff
@@ -4,8 +4,6 @@
     <!-- Don't use any sources set by the system (workaround for microbuild issue) -->
     <clear />
     <add key="nuget" value="https://www.nuget.org/api/v2/" />
-    <add key="myget.org dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="corefx lab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
```

Comment:
Do we still need this source?

Reply:
Yes, we still need it. Apparently,  we never released the microsoft.fx.portablity.* nuget packages on nuget.org, They are only available on myget.